packages/playwright-test/src/index.ts

```diff
@@ -307,7 +307,9 @@ const playwrightFixtures: Fixtures<TestFixtures, WorkerFixtures> = ({
     };
 
     const startedCollectingArtifacts = Symbol('startedCollectingArtifacts');
-    const stopTraceChunkOnContextClosure = async (tracing: Tracing) => {
+    const stopTracing = async (tracing: Tracing) => {
+      if ((tracing as any)[startedCollectingArtifacts])
```

Comment:
This will break for "reuse page between tests" scenario, because we will not collect traces for the second test due to this flag already being set.

Reply:
I don't think this change affects the behavior, so it keeps things the way they were.